Cargo.toml

```diff
@@ -17,10 +17,12 @@ default-target = "x86_64-unknown-linux-gnu"
 targets = ["i686-pc-windows-msvc", "x86_64-pc-windows-msvc", "i686-unknown-linux-gnu", "x86_64-unknown-linux-gnu", "x86_64-apple-darwin", "wasm32-unknown-unknown"]
 
 [features]
-default = ["x11", "wayland", "wayland-dlopen"]
+default = ["x11", "wayland", "wayland-dlopen", "wayland-csd-adwaita"]
 x11 = ["x11-dl", "mio", "percent-encoding", "parking_lot"]
 wayland = ["wayland-client", "wayland-protocols", "sctk"]
 wayland-dlopen = ["sctk/dlopen", "wayland-client/dlopen"]
+wayland-csd-adwaita = ["sctk-adwaita"]
+wayland-csd-title = ["sctk-adwaita/title"]
```

Comment:
I think we should make `wayland-csd-adwaita` with title and have `wayland-csd-adwaita-notitle` without it. Make `wayland-csd-adwaita` enabled by default and add a proper note in the changelog that it's enabled by default now.

I think it makes sense to have title rendering for the end users by default.

Reply:
Bump.

Reply:
Try installing `libexpat1-dev`.